core/templates/components/forms/schema-based-editors/schema-based-editor.directive.ts

```diff
@@ -67,7 +67,13 @@ angular.module('oppia').directive('schemaBasedEditor', [
       },
       template: require('./schema-based-editor.directive.html'),
       controllerAs: '$ctrl',
-      controller: [function() {}]
+      controller: [
```

Comment:
How is this change relevant to this PR (moderator page migration)?

Reply:
Yeah, to fix change detection issue with schema based editor.

Reply:
As you can see in the video, when i navigate to featured activities tab. I have to click somewhere on the page to make schema based editor visible. So, this is the fix for that.

Reply:
Without fix:
I had to click
https://user-images.githubusercontent.com/24855641/116367941-78ddf300-a825-11eb-8f17-9d47b16dddf4.mp4

With fix:
https://user-images.githubusercontent.com/24855641/116367952-7b404d00-a825-11eb-9290-6b6babad9edc.mp4